app/views/partials/income_list.scala.html

```diff
@@ -0,0 +1,38 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import ssttpaffordability.model.IncomeCategory
+
+@(income: Seq[IncomeCategory])(implicit messages: play.api.i18n.Messages)
+
+<div class="govuk-!-padding-left-6 govuk-!-margin-bottom-9">
+    <dl class="govuk-summary-list">
+
+        @{income.map( category => if (category.amount > 0)
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-font-weight-regular govuk-!-width-one-half">{messages("ssttp.affordability.your-monthly-income.form."+{s"${category.name}"})}</dt>
```

Comment:
`s"${category.name}"` is just `category.name` or `category.name.toString` 

Also, creating message keys by concatenating strings is known to go wrong over time, ie add a new income type without knowing you have to have something in messages.cy etc.  And Intellij cannot analyze the code and say that a value in. messages.cy is being used.
Worth considering something like `category.messageKey` and having the complete key stored with the item.  
( only a suggestion, nothing wrong with the code)

Reply:
Yep, that makes sense, let me change that